libcudacxx/include/cuda/functional

```diff
@@ -21,6 +21,7 @@
 #  pragma system_header
 #endif // no system header
 
+#include <cuda/__functional/address_stability.h>
```

Comment:
This feature is missing tests in `libcudacxx/test/libcudacxx/cuda/functional` as well as documentation in `docs/libcudacxx/extended_api/functional`

Reply:
At a code review meeting we discussed that it's fine to leave the tests in Thrust for now.

Documentation will be added.

Reply:
Regarding documentation, I would like to defer this for a while. I have some ideas on how we can avoid users to start sprinkling this everywhere.